src/quo2/components/avatars/wallet_user_avatar/view.cljs

```diff
@@ -0,0 +1,48 @@
+(ns quo2.components.avatars.wallet-user-avatar.view
+  (:require [quo2.components.markdown.text :as text]
+            [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.avatars.wallet-user-avatar.style :as style]
+            utils.string))
+
+(def properties
+  {:size-20 {:size        20
+             :font-size   :label
+             :font-weight :medium}
+   :size-24 {:size        24
+             :font-size   :label
+             :font-weight :semi-bold}
+   :size-32 {:size        32
+             :font-size   :paragraph-2
+             :font-weight :semi-bold}
+   :size-48 {:size        48
+             :font-size   :paragraph-1
+             :font-weight :semi-bold}
+   :size-64 {:size        64
+             :font-size   :paragraph-1
+             :font-weight :medium}
+   :size-80 {:size        80
+             :font-size   :heading-1
+             :font-weight :medium}})
+
+(defn- view-internal
+  "Options:
+
+  :full-name - string (default: nil) - used to generate initials
+  :customization-color  - keyword (default: nil) - color of the avatar
+  :size   - keyword (default: last element of properties object) - size of the
+  avatar"
+  [{:keys [full-name customization-color size theme]
+    :or   {size (last (keys properties))}}]
+  (let [circle-size (:size (size properties))
+        small?      (= size (first (keys properties)))]
```

Comment:
I caught myself putting some effort into understanding what `(last (keys properties))` and `(first (keys properties))` when reading the code.
We can add two defs after `properties` to make code more readable:
```
(def smallest-possible (first (keys properties))
(def biggest-possible (last (keys properties))
``` 
More readability, less cognitive load, no runtime evaluation) Profit)

Reply:
Makes sense. Added 👍